code/modules/mining/minebot.dm

```diff
@@ -9,7 +9,6 @@
 	icon_state = "mining_drone"
 	icon_living = "mining_drone"
 	status_flags = CANSTUN|CANWEAKEN|CANPUSH
-	stop_automated_movement_when_pulled = 1
```

Comment:
Not sure what you mean here ? There's simply the same value set twice on this type ?